app/src/main/java/io/islnd/android/islnd/messaging/Rest.java

```diff
@@ -244,4 +245,75 @@ public static void postComment(EncryptedComment encryptedComment, String apiKey)
 
         return null;
     }
+
+    public static long getServerTimeOffsetMillis(int repetitions, String apiKey) throws IOException {
+        // account for network delay by using OkHttpClient's Interceptor
```

Comment:
I think this class warrants it's own file. The benefit of extracting it is trying to keep the Rest class as simple as possible.